src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3451,6 +3483,13 @@ private TypeSymbolWithAnnotations VisitOptionalImplicitConversion(BoundExpressio
 
             (BoundExpression operand, Conversion conversion) = RemoveConversion(expr, includeExplicitConversions: false);
             var operandType = VisitRvalueWithResult(operand);
+
+            var conv = expr as BoundConversion;
+            if (conv != null && conv.ConversionKind == ConversionKind.ImplicitNullable)
+            {
+                TrackNullableStateIfNullableConversion(conv);
```

Comment:
>TrackNullableStateIfNullableConversion [](start = 16, length = 38)

Similarly, consider if it would be more appropriate to move this after the ApplyConversion call.